src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -36,11 +36,11 @@
 				<div
 					v-else
 					class="tw-w-full tw-relative tw-rounded tw-bg-cover tw-bg-center journey-card"
+					:class="{ '!tw-bg-left-top': isNonBadgeSlide(slide) }"
```

Comment:
Can we add a slide with the non-badge data to storybook?

Reply:
Oh sure, nice call!

Reply:
As struggling with component watcher from Storybook, added a ticket to follow this later.
https://kiva.atlassian.net/browse/MP-1524